### ISSUE TITLE
avatars of assigned user sometimes does not display.

### ISSUE THREAD

**Issue by treasuretron** [nextcloud/deck#2653]
On my 12" Macbook on Firefox 83 I do not see the avatar of an assigned user to a card. On my Mac Mini, same version of Firefox, avatars for assigned users are displayed on each card. Is this setting based on detected resolution of the display? It doesn't seem to be based on window size.

Anyway, would be nice to have consistency between machines as it becomes hard to parse a lot of cards at a glace without assigned-user avatars (I know I can filter but it's preferable to visually filter IMO).

**Comment by stefan-niedermann** [comment]
Please fill out the complete issue template. It ain't there for fun but helps to analyse the issue.
